tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -116,6 +116,12 @@ describeSmokeSuite(`Verifying balances consistency...`, (context) => {
         : [],
     ]);
 
+    const networkName = context.polkadotApi.runtimeChain.toString();
+    let preimages = [];
+    if (networkName === "Moonbase Stage") {
```

Comment:
This will fail for alphanet. We should most of the time use the runtime and not the network itself, because the features are implemented per runtime, not per network.